styles/style.css

```diff
@@ -20,8 +20,34 @@ body {
 body,
 button {
   font-family: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
-  color: var(--grey-dark);
+  
+  color:#292b2c;
   font-size: 1rem;
+  /* donate now button */
+  border-radius: 5px;
+  
+
+}
+/* changing 2 bottons donate now and donate bike today */
+.donate_now {
```

Comment:
It's great that you reused this class for the two similar buttons.
Another thing that is common in programming is to combine all the features of similar buttons in one class, let's say '.general-button' with all the mutual css rules in `.donate_now` and `.volunteer`. Then we add a second class on the button to define the different css rules like the `color` or `margin-left`.

Reply:
Also, when a mouse goes on any clickable item (such as button), we want to tell users to know that's clickable. For that, we convert the cursor into a hand that points to, similar to the link you have. You might want to google css cursor to learn more.

Reply:
Yes hand point cursor  i completely  missed it will definitely use it in Karma assignment. Thank you 👍